pkg/profilecreator/profilecreator.go

```diff
@@ -229,22 +229,43 @@ func (ghwHandler GHWHandler) CPU() (*cpu.Info, error) {
 	return ghw.CPU(ghwHandler.snapShotOptions)
 }
 
-// Topology returns a TopologyInfo struct that contains information about the Topology on the host system
-func (ghwHandler GHWHandler) Topology() (*topology.Info, error) {
-	return ghw.Topology(ghwHandler.snapShotOptions)
+// SortedTopology returns a TopologyInfo struct that contains information about the Topology on the host system
```

Comment:
...sorted how? Meaning, how the end result should look like? numa nodes sorted by their cell ID, increasing, then...

please add a couple lines to document this - it is part of the function contract.

Reply:
Yes, sorted by the numa ids and cpu ids, is better to add the info

Reply:
Will add a comment clarifying this.